src/pages/components/OrdinoContactForm.tsx

```diff
@@ -1,47 +1,89 @@
+        setInputs((inputs) => ({...inputs, [event.target.name]: event.target.value, mailTo: newMailTo}));
+    };
+    return {
+        inputs,
+        handleInputChange,
+    };
+};
+
+export function OrdinoContactForm() {
+
+    const {inputs, handleInputChange} = useSubmitContactForm();
+    const {name, email, subject, message, mailTo} = inputs;
 
     return (
         <Card className="shadow-sm">
             <Card.Body>
                 <Card.Text>
-                    {'Do you have questions or found a bug, do not hasitate to contact us using the contact form below. You can also contact us by writing an email to '}
-                    <Card.Link href="mailto:ordino@caleydo.org.">ordino@caleydo.org</Card.Link> or posting a message in the Microsoft Team. We are glad to help you.
+                    {'Do you have questions or found a bug, do not hesitate to contact us using the contact form below. You can also contact us by writing an email to '}
+                    <Card.Link href="mailto:ordino@caleydo.org.">ordino@caleydo.org</Card.Link>. We are glad to help you.
                  </Card.Text>
-                <Form onSubmit={() => handleSubmit()}>
+                <Form>
                     <Form.Group className="row-cols-md-3">
                         <Form.Label>Type of contact</Form.Label>
-                        <Form.Control as="select">
-                            <option>I have a question</option>
-                            <option>I want to report a bug</option>
-                            <option>Placeholder</option>
+                        <Form.Control onChange={handleInputChange} value={subject} name="subject" as="select">
+                            <option value="I have a question">I have a question</option>
+                            <option value="I want to report a bug">I want to report a bug</option>
                         </Form.Control>
                     </Form.Group>
 
                     <Form.Group>
                         <Form.Label>Message</Form.Label>
-                        <Form.Control as="textarea" rows={5} />
+                        <Form.Control as="textarea" name="message" value={message} rows={5} onChange={handleInputChange}>
+                        </Form.Control>
                     </Form.Group>
 
                     <Form.Group >
                         <Form.Label>Name</Form.Label>
-                        <Form.Control type="text" placeholder="Name" />
+                        <Form.Control value={name} onChange={handleInputChange} name="name" type="text" placeholder="Name" />
                     </Form.Group>
 
                     <Form.Group >
                         <Form.Label>Email address</Form.Label>
-                        <Form.Control type="email" placeholder="name@example.com" />
+                        <Form.Control type="email" value={email} name="email" placeholder="name@example.com" onChange={handleInputChange} />
```

Comment:
I think it's redundant having the user enter the email and name since we are opening the local client anyways.
@thinkh If you agree then I can remove this.

Reply:
Yes, I agree. We can remove both fields `name` and `email` as they are covered by the mail client.